dev/com.ibm.ws.microprofile.config.1.1/src/com/ibm/ws/microprofile/config/archaius/composite/CompositeConfig.java

```diff
@@ -30,15 +30,15 @@
 
 import com.ibm.websphere.ras.annotation.Trivial;
 import com.ibm.ws.microprofile.config.impl.ConversionManager;
-import com.ibm.ws.microprofile.config.impl.SortedSources;
 import com.ibm.ws.microprofile.config.impl.SourcedValueImpl;
+import com.ibm.ws.microprofile.config.interfaces.SortedSources;
 import com.ibm.ws.microprofile.config.interfaces.SourcedValue;
 import com.ibm.ws.microprofile.config.sources.StaticConfigSource;
 
 public class CompositeConfig implements Closeable, ConfigListener {
 
-    private final CopyOnWriteArrayList<PollingDynamicConfig> children = new CopyOnWriteArrayList<PollingDynamicConfig>();
-    private final CopyOnWriteArrayList<ConfigListener> listeners = new CopyOnWriteArrayList<ConfigListener>();
+    private final CopyOnWriteArrayList<PollingDynamicConfig> children = new CopyOnWriteArrayList<>();
+    private final CopyOnWriteArrayList<ConfigListener> listeners = new CopyOnWriteArrayList<>();
```

Comment:
Is there a reason you're not including a type for new, te field is still typed.

Reply:
It isn't needed. The `<>` operator means that the compiler can infer the type from the variable declaration.